### ISSUE TITLE
Email template error

### ISSUE THREAD

**Issue by satanasov** [ivanovd/phpBB-3.1-BG-Language#1]
Здрасти,

В превода на имейлите си допуснал малка грешка. Първия ред, запичващ с Subject: трябва да си остане Subject, а не Заглавие, защото това е реда, който се чете от скрипта за да окаже заглавие на писмото.

**Comment by ivanovd** [comment]
Здрасти. Да така е. Още при тестовете го открих това и го оправих. Явно commit-a не е минал. Сега ще recommit-на и ще пусна нова версия.

**Comment by ivanovd** [comment]
Всъщност е минал commit-a, обаче не съм го вкарал в release. Ако дръпнеш, всички файлове от Download ZIP бутона ще вземеш последната версия с доста оправени бъгчета.

**Comment by satanasov** [comment]
Аз направих clone преди 2 седмици ... и е с проблемните ... Благодаря все пак.

**Comment by ivanovd** [comment]
Нали за това говорим :)
![untitled20150520224848](https://cloud.githubusercontent.com/assets/4889583/7734925/6a21db7a-ff42-11e4-9791-3645a079b8a0.png)

**Comment by satanasov** [comment]
Да. Може и аз да съм объркал нещо и да съм копнал по-стара версия :D

**Comment by teou1** [comment]
Привет, първо мерси за превода.
Второ и аз се натъкнах на този бъг, и най-тъпото е, че първо си го дебъгнах и оправих сам и после се сетих да погледна тук да не би да откривам топлата вода:)

Само искам да допълня, да не забравяте да ъпдейтвате и зипчето на сайта на пхпбб линкнато тук - https://www.phpbb.com/customise/db/translation/bulgarian/ , защото аз мигрирах един форум на първи юни и версията, която седи там (0.5 бета) си е с този бъг. А пък проблема явно е фикснат по-отдавна, просто не е ъпдейтнат зип-а. Сочи към https://github.com/ivanovd/phpBB-3.1-BG-Language/archive/0.5-beta.zip вместо към актуалното https://github.com/ivanovd/phpBB-3.1-BG-Language/archive/master.zip

Поздрави, 
Тодор

**Comment by ivanovd** [comment]
Благодаря за отзивите! Много е полезно, когато има обратна връзка. За съжаление, все още нямам права да променям description-а на language pack-a, но ще помоля то да бъде променено. Наистина 0.5б е доста стара версия, която има не само този, но и други малки бъгчета.

**Comment by nacholibre** [comment]
Здрасти,

обнових линка до 0.6 бета, между другото, ако превода за 3.1 е готов изцяло, може направо да го кача на phpbb.com?

**Comment by ivanovd** [comment]
Здрасти, благодаря ти. Можеш ли направо към master.zip да сложиш линк?
Лятото ми остава много малко време за преводи :) Обаче поемам ангажимент август месец да имаме ако не изцяло, то на 90% ACP.

**Comment by Lio4ko** [comment]
Здравейте приятели, как върви превода ? Аз имам идея да направя един проект, но за съжаление английският ми е слаб, затова с нетърпение чакам :) Ще се радвам ако ми дадете информация за това как вървят нещата :)
Благодаря предварително.

**Comment by ivanovd** [comment]
Превода от потребителска гледна точка е 100% готов. Админ панела в момента е на 30% готов.
Можеш да свалиш последните промени от https://github.com/ivanovd/phpBB-3.1-BG-Language/archive/master.zip

**Comment by Lio4ko** [comment]
Добре, благодаря.  Тъй като не знам колко е сложно превеждането, може ли да ми кажеш приблизително, кога ще бъде на 100% преведен. (като месец например) :)

**Comment by ivanovd** [comment]
То не е сложно, но е свързано с време, което за жалост нямам в момента и ми е трудно да се ангажирам с точни дати и месеци.

**Comment by Lio4ko** [comment]
Ами добре, аз ще следя сайта, пък когато стане :)

**Comment by ivanovd** [comment]
60% от ACP е преведен. Оправени са няколк объга и съм сложил нова версия - 0.7b с последните промени.

**Comment by ivanovd** [comment]
acp/common.php е преведен изцяло ... остава още малко до финалната версия.
